test/parallel/test-http-client-aborted-event.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+const common = require('../common');
+const http = require('http');
+
+const server = http.Server(function(req, res) {
+  console.log('Server accepted request.');
+  res.write('Part of my res.');
+  res.destroy();
+});
+
+const requiredCallback = common.mustCall(function() {
+  console.log('Response aborted.');
+});
+
+server.listen(0, function() {
```

Comment:
As far as using random ports goes, most parallel tests are now using random ports. The exception is cluster tests (at least until #7043 is resolved) and one other subsystem's tests (can't recall offhand).